gson/src/test/java/com/google/gson/internal/bind/RecursiveTypesResolveTest.java

```diff
@@ -46,38 +46,42 @@
   /**
    * Test simplest case of recursion.
    */
-
-  public void testRecursiveResolveSimple() {
+  @Test
+  void testRecursiveResolveSimple() {
     @SuppressWarnings("rawtypes")
     TypeAdapter<Foo1> adapter = new Gson().getAdapter(Foo1.class);
     assertNotNull(adapter);
   }
 
-  /**
```

Comment:
Changed this here and a few lines below because they are not Javadoc comments for a specific method.